card/mod/05_email/set/right/following.rb

```diff
@@ -4,49 +4,49 @@ def virtual?; true end
 
 format :html do
 
-   view :core do |args|
-     if card.left and Auth.signed_in?
-       render_rule_editor args
-     else
-       followers_card = Card.fetch("#{card.cardname.left}+#{Card[:followers].name}") 
-       nest followers_card, :view=>:titled, :item=>:link
-     end
-   end
-   
-   view :status do |args|
-     if rcard = current_follow_rule_card
-       rcard.item_cards.map do |item|
-         %{<div class="alert alert-success" role="alert">
-           <strong>#{rcard.rule_set.follow_label}</strong>: #{item.title}
-          </div>}
-       end.join
-     else
-       "No following preference"
-     end
-   end
-   
-   view :closed_content do |args|
-     ''
-   end
-   
+  view :core do |args|
+    if card.left and Auth.signed_in?
+      render_rule_editor args
+    else
+      followers_card = Card.fetch("#{card.cardname.left}+#{Card[:followers].name}")
+      nest followers_card, view: :titled, item: :link
+    end
+  end
+
+  view :status do |args|
+    if (rcard = current_follow_rule_card)
+      rcard.item_cards.map do |item|
+        %{<div class="alert alert-success" role="alert">
+          <strong>#{rcard.rule_set.follow_label}</strong>: #{item.title}
+         </div>}
+      end.join
+    else
+      "No following preference"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.